torch/_utils.py

```diff
@@ -592,6 +592,25 @@ def dec(fun):
     return dec
 
 
+def render_call(fn, args, kwargs):
+    import torch._tensor_str
+    from torch.overrides import resolve_name
+
+    str_fn = resolve_name(fn)
+    if str_fn is None:
+        str_fn = str(fn)
+
+    def render_arg(t):
```

Comment:
Do you need this indirection for local debugging? Otherwise we cna use repr directly below

Reply:
In the old version of the patch when I wasn't changing printoptions, I had to tree_map over args/kwargs and knock out the Tensor arguments. I left this stub around in case we changed our mind. Not a big deal either way.